tests/bat/default_check_profile.sh

```diff
@@ -10,5 +10,5 @@ logfile=${LOGFILE:? LOGFILE not set}
 # Check that the logfile has correct version string
 # Remove the line in the logfile that was used for passing in KERNEL_RELEASE environment variable
 # The logfile will live at the top of the <scratchdir>/<testname> directory structure
-cat ../../$logfile | grep -v KERNEL_RELEASE | grep $kernel_release
+cat ../$logfile | grep -v KERNEL_RELEASE | grep $kernel_release
```

Comment:
Just curious -- if `$logfile` is being set by the user, why isn't it set to the correct logfilename such that you have to prefix it with `..`?

(after thinking about this for a minute) ...oh, is `$logfile` a basename?  If so, it might be easier / more robust to just set `$logfile` to an absolute pathname, and avoid exactly this kind of issue in the future.

Just a thought.

Reply:
That sounds good